From 7c8dbcca85cac601075fabf75610fc514655e329 Mon Sep 17 00:00:00 2001
From: Xu Qiang <xuqiang36@huawei.com>
Date: Fri, 3 Dec 2021 18:15:46 +0800
Subject: [PATCH] printk: Export a symbol.
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 7c8dbcca85cac601075fabf75610fc514655e329


ascend inclusion
category: bugfix
Bugzilla: N/A
CVE: N/A

-------------------------------------------

Signed-off-by: Xu Qiang <xuqiang36@huawei.com>
Acked-by: Hanjun Guo <guohanjun@huawei.com>
Signed-off-by: Yang Yingliang <yangyingliang@huawei.com>
Reviewed-by: Weilong Chen <chenweilong@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 kernel/printk/printk_safe.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel/printk/printk_safe.c b/kernel/printk/printk_safe.c
index d03c36565e0d..b774685ccf80 100644
--- a/kernel/printk/printk_safe.c
+++ b/kernel/printk/printk_safe.c
@@ -288,6 +288,7 @@ void printk_safe_flush_on_panic(void)
 
 	printk_safe_flush();
 }
+EXPORT_SYMBOL_GPL(printk_safe_flush_on_panic);
 
 #ifdef CONFIG_PRINTK_NMI
 /*
-- 
2.26.2

